<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trasmission</string>
  </property>
  <widget class="QPushButton" name="pushButton_calibrate_power">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_measured_power">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>100.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Measured power at maxpoint</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_set_input_param">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-800.000000000000000</double>
   </property>
   <property name="maximum">
    <double>800.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle/voltage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_param">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_min_param">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_max_param">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>800.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>min-max angle/voltage</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="laser_textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set power</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_set_power">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>800.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_power">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="number_points_spinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>points</string>
   </property>
  </widget>
  <zorder>label_11</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>pushButton_calibrate_power</zorder>
  <zorder>doubleSpinBox_measured_power</zorder>
  <zorder>label_2</zorder>
  <zorder>doubleSpinBox_set_input_param</zorder>
  <zorder>pushButton_set_param</zorder>
  <zorder>doubleSpinBox_min_param</zorder>
  <zorder>doubleSpinBox_max_param</zorder>
  <zorder>laser_textBrowser</zorder>
  <zorder>doubleSpinBox_set_power</zorder>
  <zorder>pushButton_set_power</zorder>
  <zorder>number_points_spinBox</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
